subsys/net/lib/http/http_client.c

```diff
@@ -279,8 +195,9 @@ static int http_client_request(struct http_ctx *ctx)
 	ctx->rsp_buf = malloc(HTTP_RESPONSE_MAX_SIZE);
 	if (ctx->rsp_buf < 0) {
```

Comment:
I don't think it's working here.

Reply:
could you help tell me why~！which lines?

Reply:
Refer to prototype.
void *malloc(size_t size);
You could get NULL retval if it failed to allocate memory.

Reply:
Got it!